test/fixtures/linter/rules/NoGlobals/NoExportedLibValues.js

```diff
@@ -0,0 +1,16 @@
+sap.ui.define(
+	["sap/ui/unified/library", "sap/ui/core/library", "sap/ui/comp/library"],
+	function (unifiedLibrary, coreLibrary, compLibrary) {
+		"use strict";
+
+		var SelectOptionSign = compLibrary.smartfilterbar.SelectOptionSign;
```

Comment:
Shouldn't this then be part of the `_Negative` fixture then, as it's not a finding and the correct usage?